CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v0.26.0 (Jun 10, 2021)
+
+IMPROVEMENTS:
+* Arm CPUs no longer need special builds [[GH-1471](https://github.com/hashicorp/consul-template/pull/1471), [GH-1404](https://github.com/hashicorp/consul-template/issues/1404)]
+* add 'md5sum' template function [[GH-1464](https://github.com/hashicorp/consul-template/pull/1464)]
+* add 'envOrDefault' template function [[GH-1461](https://github.com/hashicorp/consul-template/pull/1461), [GH-829](https://github.com/hashicorp/consul-template/issues/829)]
+* configurable Vault default lease duration[[GH-1446](https://github.com/hashicorp/consul-template/pull/1446), [GH-1445](https://github.com/hashicorp/consul-template/issues/1445)]
```

Comment:
nit: option to include a space between 'duration' and the GitHub links

Reply:
Great eye for detail. Fixed this and the other spacing issue. Thanks!